lib/ui/page/home/introduction/view.dart

```diff
@@ -177,7 +177,7 @@ class IntroductionView extends StatelessWidget {
                             key: const Key('NumCopyable'),
                             text: c.num.text,
                             label: 'label_num'.l10n,
-                            share: 'Gapopa ID: ${c.myUser.value?.num.val}',
+                            share: 'Gapopa ID: ${c.myUser.value?.num.toString}',
```

Comment:
Тут пропустили `()`, чтобы это был вызов функции, но на самом деле внутри `'${ ... }'`, насколько я знаю, необязательно вызывать `toString()` - `Dart` автоматически его вызовет:

```suggestion
                            share: 'Gapopa ID: ${c.myUser.value?.num}',
```

Проверьте, пожалуйста?

Reply:
Да, верно. Поправил